### ISSUE TITLE
Widget

### ISSUE THREAD

**Issue by Angelk90** [waseefakhtar/dose-android#124]
Hi @waseefakhtar , It would be interesting if there were widgets that could be placed on the home page.

**Comment by waseefakhtar** [comment]
I agree! I thought about it too to have it at some point that shows the next medication with the time remaining or a list of the next few medications but feel free to suggest how you imagine it to be. :)
